lib/govuk_publishing_components/presenters/public_layout_helper.rb

```diff
@@ -46,17 +46,48 @@ class PublicLayoutHelper
       attr_reader :footer_navigation, :footer_meta, :cookie_banner_data
 
       def initialize(local_assigns)
+        @footer_links_count = generate_ga4_footer_links_count(local_assigns)
         @footer_navigation = local_assigns[:footer_navigation] || navigation_link_generation_from_locale(I18n.t("components.layout_footer.navigation_links"))
-        @footer_meta = local_assigns[:footer_meta] || { items: add_data_attributes_to_links(FOOTER_META[:items], "supportLink") }
+        @footer_meta = local_assigns[:footer_meta] || { items: add_data_attributes_to_links(FOOTER_META[:items], "supportLink", { section: "Support links", index_section: 3 }) }
         @cookie_banner_data = local_assigns[:cookie_banner_data] || {}
       end
 
+      def generate_ga4_footer_links_count(local_assigns)
```

Comment:
Just looping through the `Topics`, `Government Activity`, and `Meta` links and totalling the count of them. I then `add 2` at the end to account for the `OGL` and `Crown Copyright` link.